lib/less/parser/parser.js

```diff
@@ -390,19 +390,51 @@ var Parser = function Parser(context, imports, fileInfo) {
                     return new(tree.Call)(name, args, index, fileInfo);
                 },
                 arguments: function () {
-                    var args = [], arg;
+                    var argsSemiColon = [], argsComma = [],
+                        expressions = [],
+                        isSemiColonSeparated, value, arg;
+
+                    parserInput.save();
 
                     while (true) {
```

Comment:
Thanks... Yes, it seems to need more investigations on possible unifications.
I was just about to experiment with some `@var(args...);` stuff, and realized that having three distinct functions to handle pretty much the same syntax is somewhat an overkill... Additionally I recall that for #1857, I was also thinking of allowing `@arg...` stuff for functions too, but quickly bailed out because of all these differences :(